src/NServiceBus.NHibernate/FodyWeavers.xml

```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Weavers>
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Weavers GenerateXsd="false">
```

Comment:
Probably not inteded?

Reply:
yes that's needed otherwise it's going to generate the XSD every time you build